gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginRegistryImpl.java

```diff
@@ -68,32 +73,21 @@ public class PluginRegistryImpl extends AbstractService implements PluginRegistr
         PLUGIN_TYPE_PROPERTY_ALIASES.put("service_discovery", "service-discoveries");
     }
 
-    @Autowired
-    private PluginRegistryConfiguration configuration;
+    private final PluginRegistryConfiguration configuration;
 
-    @Autowired
-    private Environment environment;
+    private final Environment environment;
 
-    @Autowired
-    @Qualifier("corePluginExecutor")
-    private ExecutorService executor;
+    private final ExecutorService executor;
```

Comment:
There was a @Qualifier("corePluginExecutor") on the previous authowed, are you should the right executor is injected here?

Reply:
yep, I use it in the @Configuration class